server/fleet/hosts.go

```diff
@@ -464,6 +468,11 @@ type MDMHostData struct {
 	// host-returning methods.
 	DeviceStatus  *string `json:"device_status,omitempty" db:"-" csv:"-"`
 	PendingAction *string `json:"pending_action,omitempty" db:"-" csv:"-"`
+
+	// ConnectedToFleet indicates if the host has an active MDM connection
+	// with this Fleet instance. This boolean is not filled by all
+	// host-returning methods.
+	ConnectedToFleet *bool `json:"connected_to_fleet" csv:"-" db:"connected_to_fleet"`
```

Comment:
Is this filter meant to be surfaced to end users in the UI?

Reply:
currently only used in fleetctl